webpack.common.config.js

```diff
@@ -1,14 +1,13 @@
 /* global module __dirname */
 
 var path      = require('path');
-var EMPTY     = 'widget/empty';
 var TARGET_JS = path.resolve(__dirname, 'target/js/');
-var SHARED_JS = TARGET_JS + '/shared';
+var LOCAL_PACKAGES = path.resolve(__dirname, 'packages/');
 
 // Return a function so that all consumers get a new copy of the config
 module.exports = function (outputFilename) {
   return {
-    entry: ['./src/util/RegisterInputs.js', './target/js/widget/OktaSignIn.js'],
```

Comment:
do we still need https://github.com/okta/okta-signin-widget/blob/master/src/util/RegisterInputs.js ? Since you are removing it here and I see that packages/@okta/courage-for-signin-widget/src/CourageForSigninWidget.js already does the same, I thought it was worth asking

Reply:
ah, good point. I don't think we need it.